challenges/objects-arrays.js

```diff
@@ -1,88 +1,220 @@
-  { id: 2, first_name: "Saundra", university: "The School of the Art Institute of Chicago", email: "swhal1@state.gov" },
-  { id: 3, first_name: "Lambert", university: "Marian College", email: "lparham2@techcrunch.com" },
-  { id: 4, first_name: "Modestine", university: "International Medical & Technological University", email: "mdolder3@symantec.com" },
-  { id: 5, first_name: "Chick", university: "Sultan Salahuddin Abdul Aziz Shah Polytechnic", email: "camorts4@google.com.au" },
-  { id: 6, first_name: "Jakob", university: "Fachhochschule Rosenheim, Hochschule für Technik und Wirtschaft", email: "jharken5@spiegel.de" },
-  { id: 7, first_name: "Robbi", university: "Salem University", email: "rbrister6@redcross.org" },
-  { id: 8, first_name: "Colline", university: "Coastal Carolina University", email: "cbrosh7@alibaba.com" },
-  { id: 9, first_name: "Michail", university: "Universidad Católica de Ávila", email: "mrome8@shinystat.com" },
-  { id: 10, first_name: "Hube", university: "Universitat Rovira I Virgili Tarragona", email: "hlethbrig9@foxnews.com" },
+  {
+    id: 1,
+    first_name: "Cynde",
+    university: "Missouri Southern State College",
+    email: "ctorry0@macromedia.com"
+  },
+  {
+    id: 2,
+    first_name: "Saundra",
+    university: "The School of the Art Institute of Chicago",
+    email: "swhal1@state.gov"
+  },
+  {
+    id: 3,
+    first_name: "Lambert",
+    university: "Marian College",
+    email: "lparham2@techcrunch.com"
+  },
+  {
+    id: 4,
+    first_name: "Modestine",
+    university: "International Medical & Technological University",
+    email: "mdolder3@symantec.com"
+  },
+  {
+    id: 5,
+    first_name: "Chick",
+    university: "Sultan Salahuddin Abdul Aziz Shah Polytechnic",
+    email: "camorts4@google.com.au"
+  },
+  {
+    id: 6,
+    first_name: "Jakob",
+    university:
+      "Fachhochschule Rosenheim, Hochschule für Technik und Wirtschaft",
+    email: "jharken5@spiegel.de"
+  },
+  {
+    id: 7,
+    first_name: "Robbi",
+    university: "Salem University",
+    email: "rbrister6@redcross.org"
+  },
+  {
+    id: 8,
+    first_name: "Colline",
+    university: "Coastal Carolina University",
+    email: "cbrosh7@alibaba.com"
+  },
+  {
+    id: 9,
+    first_name: "Michail",
+    university: "Universidad Católica de Ávila",
+    email: "mrome8@shinystat.com"
+  },
+  {
+    id: 10,
+    first_name: "Hube",
+    university: "Universitat Rovira I Virgili Tarragona",
+    email: "hlethbrig9@foxnews.com"
+  }
 ];
 
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array. This will be an array of strings.
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
+const universities = graduates
+  .map(graduate => {
+    return graduate.university;
+  })
+  .sort();
```

Comment:
In case you didn't already know, you could remove the curly brackets and have it all as a one liner

`const universities = graduates.map(graduate =>  graduate.university) .sort()`

No need for curly brackets and return keyword when it can be all done in one line. Personal preference I guess of what looks neat vs whats more readable I guess.